src/fees/FeeGenerator.sol

```diff
@@ -103,27 +124,6 @@ abstract contract FeeGenerator is Ownable {
         return fees;
     }
 
-    /// @notice Set fee calculator contracts
-    function setFeeCalculators(
-        bytes4[] calldata selectors,
-        address[] calldata tos,
-        address[] calldata feeCalculators_
-    ) external onlyOwner {
-        uint256 length = selectors.length;
-        if (length != tos.length) revert LengthMismatch();
-        if (length != feeCalculators_.length) revert LengthMismatch();
-
-        for (uint256 i = 0; i < length; ) {
-            bytes4 selector = selectors[i];
-            address to = tos[i];
-            address feeCalculator = feeCalculators_[i];
-            setFeeCalculator(selector, to, feeCalculator);
-            unchecked {
-                ++i;
-            }
-        }
-    }
-
     function setFeeCalculator(bytes4 selector, address to, address feeCalculator) public onlyOwner {
         feeCalculators[selector][to] = feeCalculator;
         emit FeeCalculatorSet(selector, to, feeCalculator);
```

Comment:
Relocate this function to align with external `setFeeCalculators`

Reply:
fixed